examples/community/README.md

```diff
@@ -1222,7 +1222,7 @@ The resulting images in order:-
 ![result5](https://huggingface.co/datasets/NagaSaiAbhinay/UnCLIPImageInterpolationSamples/resolve/main/starry_to_flowers_5.png)
 
 ### DDIM Noise Comparative Analysis Pipeline
-#### **Research question: What visual concepts do the diffusion models learn from each noise level during training?**
+#### **Research question: What visual concepts do the diffusion models learn from each noise level during training?**
```

Comment:
Are changes in this file meant to be in this PR? Is this a leftover from Img2Vid?

Reply:
Ah, seems like so. I think my editor is to blame :smiling_face_with_tear:

Reply:
I'm not sure why even after resolving merge confict with main and applying the changes from there it doesn't fix